### PR TITLE
Markdown pages set up as content source

### PR DESCRIPTION
**Pull request by aannyyaa**

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  siteMetadata: {
+    title: "My Gatsby Site",
```
Comment (saifulislamdev) [inline gatsby-config.js]: Give it a more meaningful name.

File: gatsby-config.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
   siteMetadata: {
-    title: "My Gatsby Site",
+    title: "LPaaS 2.0",
```
Comment (saifulislamdev) [inline gatsby-config.js:3]: Never mind, see you do that here.

File: README.md [hunk 3]
```diff
@@ -1,13 +1,21 @@
...
-  |- CODE_OF_CONDUCT.md 
+  |- CODE_OF_CONDUCT.md
   |- ISSUE_TEMPLATE
   |- LICENCE.md
   |- PULL_REQUEST_TEMPLATE.md
+
+  ## How to run the template site on your local computer
+
+  1. Clone the repository `git clone https://github.com/nyc-cto/LPaaS2.git`
+  1. Navigate into the directory `cd LPaaS2`
```
Comment (saifulislamdev) [inline README.md:19]: Number these 1-4.

Comment (aannyyaa) [reply README.md:19]: They are automatically numbered by markdown

File: README.md [hunk 4]
```diff
@@ -12,10 +12,10 @@ In this application:
   |- LICENCE.md
   |- PULL_REQUEST_TEMPLATE.md
 
+```
```
Comment (saifulislamdev) [inline README.md:15]: It looks like you're wrapping everything in one code block, which I would not suggest.

Comment (aannyyaa) [reply README.md:15]: that's from the initial readme that Justin wrote. I only wrote lines 16+

**Pull request merged**